packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -520,14 +520,14 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
 	protected async initializingFirstTime() {
 		this.root.set(taskManagerKey, TaskManager.create(this.runtime).handle);
 		// Create virtual data store
```

Comment:
@tyler-cai-microsoft Could you add a comment here explaining why the code is commented out.